scripts/west_commands/runners/jlink.py

```diff
@@ -35,6 +35,7 @@ class JLinkBinaryRunner(ZephyrBinaryRunner):
 
     def __init__(self, cfg, device, dev_id=None,
                  commander=DEFAULT_JLINK_EXE,
+                 select="usb",
```

Comment:
I think we should name this `transport` instead, because we may want to generalize this in the future, given that several debuggers support this.